MANIFEST.in

```
@@ -1,5 +1,5 @@
 include README.rst CHANGES LICENSE
-include test_responses.py test_matchers.py test_registries.py
+include responses/tests/*.py
```

Considering that this is not the first time when tests are forgotten more safe might be to extend wildcard
```suggestion
include **/test_*.py
```